### PR TITLE
(#5) - sum of odd array elements

### PR DESCRIPTION
**Pull request by ghost**
Task 1 ready to go

### CODE REVIEW

**Comment by neojski** [comment]
Last commit contains two modified files which look identical. You probably don't want to modify ex3.js. It also looks like `f` can take a number as its argument. From the description I wouldn't guess that `f(1)` is legal.

But tests and the general ideal looks perfect.

**Comment by ghost** [comment]
Look at result. There is no ex3.js file. I can't explain what git has done with my branch.

**Comment by ghost** [comment]
OK. There IS ex3.js file (which was deleted... but then git forced me to merge with origin). We talked about this...

File: tasks/homework/sum-of-odd-numbers.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            notStrictEqual(f, undefined, 'f is defined');
+            strictEqual(typeof f, 'function', 'f is a function');
+
+            var res = [];
+            var ans = 0;
+            for (var i = 0; i < 100; i++) {
+                res[i] = i;
+                if (i % 2 === 1)
+                    ans += i;
+                strictEqual(f(i), ans, 'correct value of f(' + i + ')');
```
Comment (neojski) [inline tasks/homework/sum-of-odd-numbers.js:18]: You probably wanted here `f(res)`

**Comment by neojski** [comment]
Your tests are broken. I'll fix them but always be sure that your original program passes the tests before you submit.

**Comment by neojski** [comment]
Fixed, squashed and merged in 2b3583459a73aad049b68421199eb5f137829998

**Comment by neojski** [comment]
I really like this task, cheers!

**Pull request closed without merging**